hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -710,6 +765,26 @@ private Pair<Option<String>, JavaRDD<WriteStatus>> writeToSink(JavaRDD<HoodieRec
             + totalErrorRecords + "/" + totalRecords);
       }
       String commitActionType = CommitUtils.getCommitActionType(cfg.operation, HoodieTableType.valueOf(cfg.tableType));
```

Comment:
I don't think we need to address this right away, but lets think if it makes sense.
for error records from WriteStatusRdd, should we also route them to error table. I vaguely remember a user from the community asking, if hudi can ignore such error records and commit rest of the valid data long time back.
and since we are adding support for error table, may be we should consider routing failed records to the error table as well.

Reply:
Created https://issues.apache.org/jira/browse/HUDI-5858 for it.